test/test_latlng.rb

```diff
@@ -254,4 +254,18 @@ def test_to_dms
     assert_equal [-87, 39], dms[0, 2]
     assert_equal 37, dms[2].round
   end
+
+  def test_invalid_units
+    expected_exception_message = 'feet is an unsupported unit of length.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.